spotbugs/src/main/java/edu/umd/cs/findbugs/ba/FieldSummary.java

```diff
@@ -199,28 +199,20 @@ public void mergeSummary(XField fieldOperand, OpcodeStack.Item mergeValue) {
      *            The complete to set.
      */
     public void setComplete(boolean complete) {
-        int fields = 0;
-        int removed = 0;
-        int retained = 0;
         this.complete = complete;
         if (isComplete()) {
             for (Iterator<Map.Entry<XField, OpcodeStack.Item>> i = summary.entrySet().iterator(); i.hasNext();) {
                 Map.Entry<XField, OpcodeStack.Item> entry = i.next();
                 XField f = entry.getKey();
                 if (AnalysisContext.currentXFactory().isReflectiveClass(f.getClassDescriptor())) {
                     i.remove();
-                    removed++;
                     continue;
                 }
                 OpcodeStack.Item defaultItem = new OpcodeStack.Item(f.getSignature());
-                fields++;
                 Item value = entry.getValue();
                 value.makeCrossMethod();
                 if (defaultItem.equals(value)) {
                     i.remove();
-                    removed++;
-                } else {
-                    retained++;
```

Comment:
this looks strange, assuming there was some intention to handle the (unused) fields.